com_dpfields/admin/helpers/dpfields.php

```diff
@@ -187,6 +197,14 @@ public static function getFields ($context, $item = null, $prepareValue = false)
 						));
 					}
 
+					/*
+					 * On after field render
+					 * Event allow plugins to modfify the output of the prepared field
+					 */
+					if ($value)
```

Comment:
No I think the value should not be empty. Because  I can see some plugins using this event will not even check if it is empty!
I think in general it should not be empty.